### DISCUSSION TITLE
Anyway to get the screen to dim brightness when not being used?

### DISCUSSION THREAD

**Discussion by Jonnyjoebo** [Desuuuu/klipper#159] [Q&A] [answered]
The screen used to auto-dim itself if you didn't tap for a bit. That could be with the stock firmware only. Does anyone know about this on the DGUS-reloaded-Klipper firmware? Thanks:)

**Comment by Desuuuu** [comment, marked as answer]
It can be configured using the `T5UID1.cfg` file but I suspect the standby brightness setting would be overridden on boot by the screen implementation in klippy, so it would end up not working.

This would require modifications to klippy in order to work I guess.
